cocos/editor-support/cocostudio/CCActionObject.cpp

```diff
@@ -192,7 +192,7 @@ bool ActionObject::valueToBool(const std::string& value)
 }
 float ActionObject::valueToFloat(const std::string& value)
 {
-    return utils::atof(value.c_str());
+    return (float)utils::atof(value.c_str());
```

Comment:
Why need to convert `atof` to float? Isn't `atof` return float?

Reply:
the `atof` convert string to double.